src/plugins/intel_cpu/src/ngraph_transformations/align_eltwise_input_ranks.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+/*
+ * Description:
+ * transformation aligns elementwise input ranks by adding unsqueeze to constant node that has smaller rank than the other input
+ */
```

Comment:
Please change a description according to new changes (we won't insert unsqueeze node).

Reply:
And please use doxygen comments style